libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -938,28 +936,17 @@ private void removeAccount(ClientManager clientMgr, final boolean showLoginPage,
     				for (int i = 0; i < accounts.length - 1; i++) {
     					clientMgr.removeAccounts(accounts);
     				}
-    				clientMgr.removeAccountAsync(accounts[accounts.length - 1],
-    						new AccountManagerCallback<Boolean>() {
-
-    	    			@Override
-    	    			public void run(AccountManagerFuture<Boolean> arg0) {
-    	    				notifyLogoutComplete(showLoginPage);
-    	    			}
-    	    		});
+    				clientMgr.removeAccount(accounts[accounts.length - 1]);
+                    notifyLogoutComplete(showLoginPage);
```

Comment:
If we call this in the positive and ever single negative (4 times total) why not just put it after the conditional line ~951?

Reply:
Good catch. Will do.